src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -30,7 +30,7 @@
 # CUSTOM CHOICE LISTS
 
 secret_encoding_values = secret_text_encoding_values + secret_binary_encoding_values
-certificate_format_values = ['PEM', 'DER']
+key_format_values = certificate_format_values = ['PEM', 'DER']
```

Comment:
why delete certificate_format_values?

Reply:
@yungezz It is not deleted, I just change `certificate_format_values = ['PEM', 'DER']` to `key_format_values = certificate_format_values = ['PEM', 'DER']`.